Description: Do not crash on homedir creation
 Glanceclient just crashes if it can't mkdir("~"). This fixes it.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-04-20

--- python-glanceclient-0.17.0.orig/glanceclient/shell.py
+++ python-glanceclient-0.17.0/glanceclient/shell.py
@@ -556,7 +556,10 @@ class OpenStackImagesShell(object):
     def _cache_schemas(self, options, home_dir='~/.glanceclient'):
         homedir = expanduser(home_dir)
         if not os.path.exists(homedir):
-            os.makedirs(homedir)
+            try:
+                os.makedirs(homedir)
+            except Exception:
+                pass
 
         resources = ['image', 'metadefs/namespace', 'metadefs/resource_type']
         schema_file_paths = [homedir + os.sep + x + '_schema.json'
